IBPSA/BoundaryConditions/WeatherData/BaseClasses/EquationOfTime.mo

```diff
@@ -27,6 +28,11 @@ This component computes the difference between solar noon and noon of local civi
 </html>", revisions="<html>
 <ul>
 <li>
+March 27, 2023, by Ettore Zanetti:<br/>
+Updated to use partial class for conversion from simulation time to calendar time.<br/>
+This is for <a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1716\">IBPSA #1716</a>.
+</li>
+<li>
```

Comment:
I noticed that there is a typo in the documentation section of this model. It mentions "local civi**c** time", I think this should be "local civil time". Could you also fix this?